samples/file-encryptor/shared.h

```diff
@@ -7,17 +7,20 @@
 #define HASH_VALUE_SIZE_IN_BYTES 32
 #define ENCRYPTION_KEY_SIZE 256
 #define ENCRYPTION_KEY_SIZE_IN_BYTES (ENCRYPTION_KEY_SIZE / 8)
+#define SALT_SIZE_IN_BYTES 16
 
 // encryption_header_t contains encryption metadata used for decryption
 // file_data_size: this is the size of the data in an input file, excluding the
-// header digest: this field contains hash value of a password encrypted_key:
-// this is the encrypted version of the encryption key used for encrypting and
-// decrypting the data
+// header digest: this field contains hash value of a password
+// encrypted_key: this is the encrypted version of the encryption key used for
+//                encrypting and decrypting the data
+// salt: The iv/ salt used in PBKDF and AES encryption/ decryption
```

Comment:
Wording for clarity:
```suggestion
// salt: The salt value used in deriving the password key.
//       It is also used as the IV for the encryption/decryption of the data.
```

```

Reply:
Incorporated suggestion.